packages/engine-ui/src/components/Map.js

```diff
@@ -33,7 +33,8 @@ const Map = ({ data }) => {
   const layers = [
     mapUtils.toIconLayer(
       mapUtils.vehicleIcon(data.vehicles),
-      UIState.highlightTransport
+      UIState.highlightTransport,
+      'vehicle'
```

Comment:
I think in discussions that we said we should call it `transport`, right?

Reply:
Goes for whole PR

Reply:
Should I just swap vehicle for tarnsport throughout the whole application?